File: functions/Finantial_calculator_update.py
# Douglas london, finantial calculator Update Pyhton

print("Hello, this is my finantial calculator that calculates savings, spendins, and other finantial expenses.")

def collecting_data(type):
    return input(f"What is your monthly {type} cost:\n")
 
def info(cost, income, type):
    percent = cost/income*100
    print(f"Your {type} is ${cost:.2f}, which is {percent}% of your income.")


    
income = float(input("What is your monthly income:\n"))
rent = float(collecting_data("rent"))
utilities = float(collecting_data("utilities"))
groceries = float(collecting_data("groceries"))
transportation = float(collecting_data("transportation"))

savings = income*0.1
spendings = income-savings-rent-utilities-groceries-transportation

info(rent, income, "rent")
info(utilities, income, "utillities")
info(rent, income, "groceries")
info(rent, income, "transportation")
info(savings, income, "savings")
info(spendings, income, "spendings")

File: functions/Finantial_calculator_update.c
// Douglas London, Finantial calculator update C
#include <stdio.h>

float inputs(char type[20], int input[50]){
    printf("What is your monthly %s cost:\n", type);
    scanf("%f", &input);
    return;
}

float income;
float utilities;
float groceries;
float transportation;


int main(void){
    
printf("Hello, this is my finantial calculator that calculates savings, spendins, and other finantial expenses.\n");
printf("What is your monthly income:\n");
scanf("%f", income);

float rent = inputs("rent", 0);








    return 0;
}